### ISSUE TITLE
About post-processing to map location to bounding box

### ISSUE THREAD

**Issue by qiaodongxing** [Awenbocc/GEMeX-Project#2]
This is a great piece of work and the methods in this paper are of great interest. One of the papers mentions a post-processing method that mapping location to bounding box, can I ask the authors what implementation is used for this operation and what exactly is this post-processing

**Comment by Awenbocc** [comment]
Hi, thanks for your interest! Specifically, each image contains approximately 30 anatomical regions (including the bounding boxes obtained through detection). Upon receiving the anatomical region output from GPT-4o (in text format), we map this text to the pre-extracted bounding box coordinates.

**Comment by qiaodongxing** [comment]
Thank you for your timely reply, my understanding is that each image is pre-obtained by detecting the model with the bounding box of 30 anatomical regions and the post-processing process is to visualize the anatomical regions involved in the vqa, am I correct?

**Comment by Awenbocc** [comment]
> Thank you for your timely reply, my understanding is that each image is pre-obtained by detecting the model with the bounding box of 30 anatomical regions and the post-processing process is to visualize the anatomical regions involved in the vqa, am I correct?

That's correct! Equipping textually anatomical regions with visual locations ([x_min, y_min, width, height])for visualization.
